cli/scripts/start-build.js

```diff
@@ -22,6 +22,10 @@ includeTypes.forEach((folder) => {
   shell.cp('-R', source, 'build/types')
 })
 
+// TODO: Add a typescript or rollup build step
```

Comment:
Is there a github issue for this we can link? Have been trying to do `TODO: blah, https://github.com/blahblah`.

Reply:
There isn't right now.